ddtrace/contrib/logging/patch.py

```diff
@@ -77,20 +75,17 @@ def _w_makeRecord(func, instance, args, kwargs):
     # logs from internal logger may explicitly pass the current span to
     # avoid deadlocks in getting the current span while already in locked code.
     span_from_log = getattr(record, _LOG_SPAN_KEY, None)
+    trace_details = {}
     if isinstance(span_from_log, ddtrace.Span):
         span = span_from_log
     else:
-        span = _get_current_span(tracer=config.logging.tracer)
-
-    if span:
-        trace_id = span.trace_id
-        if config._128_bit_trace_id_enabled and not config._128_bit_trace_id_logging_enabled:
-            trace_id = span._trace_id_64bits
-        setattr(record, RECORD_ATTR_TRACE_ID, str(trace_id))
-        setattr(record, RECORD_ATTR_SPAN_ID, str(span.span_id))
-    else:
-        setattr(record, RECORD_ATTR_TRACE_ID, RECORD_ATTR_VALUE_ZERO)
-        setattr(record, RECORD_ATTR_SPAN_ID, RECORD_ATTR_VALUE_ZERO)
+        tracer = _get_tracer(tracer=config.logging.tracer)
+        if tracer:
+            span = tracer.current_span()
+            trace_details = tracer.get_log_correlation_context(active=span)
```

Comment:
Do we need this? Can we replace this line with and remove the `active` argument from `get_log_correlation_context`.

```suggestion
            trace_details = tracer.get_log_correlation_context()
```

Reply:
removed